utils/report.c

```diff
@@ -175,10 +178,10 @@ static struct sort_key sort_func = {
 	.list = LIST_HEAD_INIT(sort_func.list),
 };
 
-static struct sort_key *all_sort_keys[] = {
-	&sort_total,	&sort_total_avg, &sort_total_min, &sort_total_max, &sort_self,
-	&sort_self_avg, &sort_self_min,	 &sort_self_max,  &sort_call,	   &sort_func,
-};
+static struct sort_key *all_sort_keys[] = { &sort_total,     &sort_total_avg, &sort_total_min,
+					    &sort_total_max, &sort_self,      &sort_self_avg,
+					    &sort_self_min,  &sort_self_max,  &sort_call,
+					    &sort_func,	     &sort_size };
```

Comment:
Please follow the existing coding style like
```
static struct sort_key *all_sort_keys[] = {
   ...
};
```

The open and closing braces are on separate lines.

Reply:
pre-commit automatically modify separate lines to one line.

```
Check Yaml...........................................(no files to check)Skipped
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
isort................................................(no files to check)Skipped
clang-format.............................................................Failed
- hook id: clang-format
- files were modified by this hook
fistki@uftrace ~/func-size(func-size)$ git diff
diff --git a/utils/report.c b/utils/report.c
index 62a8b94d..834cec22 100644
--- a/utils/report.c
+++ b/utils/report.c
@@ -181,8 +181,7 @@ static struct sort_key sort_func = {
 static struct sort_key *all_sort_keys[] = { &sort_total,     &sort_total_avg, &sort_total_min,
                                            &sort_total_max, &sort_self,      &sort_self_avg,
                                            &sort_self_min,  &sort_self_max,  &sort_call,
-                                           &sort_func,      &sort_size
-};
+                                           &sort_func,      &sort_size };
```
What should I do?

Reply:
Let's keep it as is. We should find if there is a such rule in clang-format configs.
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

I can't find any so far.

Reply:
Hmm.. I'm a bit surprised that they don't have 'Linux' base style yet.

Reply:
It looks like they don't have a specific option for this.  But I also found that trailing comma (after `&sort_size` in this case) could do the trick.

https://stackoverflow.com/questions/47409797/how-to-wrap-struct-initializer-in-clang-format

Reply:
It works. Thanks!